Dockerfile

```diff
@@ -16,6 +16,7 @@ RUN rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-fedora-28-x86_64
 
 RUN dnf -y update
 # python2-pip is not available on F25.
+# rpm-devel is needed by rpm-py-installer.
```

Comment:
You mean like this way?
Because I am not sure that the below format is valid or not.

```
RUN dnf -y install \
...
  python-pip \ # python2-pip is not available on F25.
...
  rpm-devel \ # needed by rpm-py-installer.
  && dnf clean all
```

Reply:
More like this:
```
...
  # python2-pip is not available on F25
  python-pip \
...
  # needed by rpm-py-installer
  rpm-devel \
...
```
I think it's allowed in Dockerfile, but I'm not sure.

Reply:
After trying put comment on the right side of the related package, I got a parse error.

> https://travis-ci.org/rebase-helper/rebase-helper/builds/275485898
> Error response from daemon: Dockerfile parse error line 46: unknown instruction: &&